index.html

```diff
@@ -73,7 +73,34 @@ <h2>While you wait...</h2>
       <div id="canvasContainer"></div>
 
       <h2 id="signup">Sign up</h2>
-      <p>TODO</p>
+      <form>
+      <div class="item">
+        <input type="text" name="name" class="question" id="name" required autocomplete="off" />
+        <label for="name"><span>What's your name?</span></label>
+      </div>
+
+      <div class="item">
+        <input type="text" name="email" class="question" id="email" required autocomplete="off" />
+        <label for="email"><span>What's your email?</span></label>
+      </div>
+
+      <div class="item">
+        <input type="text" name="location" class="question" id="location" required autocomplete="off" />
+        <label for="location"><span>Where are you hacking from?</span></label>
+      </div>
+
+      <div class="item">
+        <input type="text" name="cats" class="question" id="cats" required autocomplete="off" />
+        <label for="cats"><span><i>Optional:</i> Thought on the movie Cats (2019)?</span></label>
+      </div>
+
+
+      <p style="text-align: center"><a class="button" href="https://forms.gle/mPmo9hbThWfjDeuz5" id="apply">Apply</a></p>
+
+      <p style="text-align: center">...and then check out the <a href="https://www.facebook.com/events/316698152296365/">Facebook event!</a></p>
+
+      </form>
+    </section>
```

Comment:
I think this `</section>` doesn't have a matching opening tag, was there supposed to be one?

Reply:
oops its gone now